scripts/travis-extract-l10n.sh

```diff
@@ -0,0 +1,165 @@
+#! /bin/bash
+# This script is supposed to be running automatically via travis cronjobs.
+# It's also possible to be run manually if necessary.
+#
+# This script will do the following:
+#   - prepare git credentials for pull request push
+#   - create a new branch (l10n-extract-2017-08-24-0b3bcaf2ca)
+#   - Update your code
+#   - Extract new strings and push to the .po files
+#
+# If you're running the script manually please make sure to expose
+# the following variables to the environment:
+#   - GITHUB_TOKEN (to the github token of addons-robot,
+#                   talk to tofumatt or cgrebs)
+#   - TRAVIS_REPO_SLUG="mozilla/addons-server"
+#   - TRAVIS_BRANCH="master"
+
+set -o errexit -o nounset
+
+REV=$(git rev-parse --short HEAD)
+MESSAGE="Extracted l10n messages from $(date -u --iso-8601=date) at $REV"
+BRANCH_NAME="l10n-extract-$(date -u --iso-8601=date)-$REV"
+ROBOT_EMAIL="addons-dev-automation+github@mozilla.com"
+ROBOT_NAME="Mozilla Add-ons Robot"
+
+# gettext flags
+CLEAN_FLAGS="--no-obsolete --width=200 --no-location"
+MERGE_FLAGS="--update --width=200 --backup=none"
+UNIQ_FLAGS="--width=200"
+DEBUG_LOCALES="dbl dbr"
+
+GIT_USER_EMAIL="$(git config --global user.email)"
+GIT_USER_NAME="$(git config --global user.name)"
+GIT_SIGNING="$(git config --global commit.gpgsign)"
+GIT_SIGNING_KEY="$(git config --global user.signinkey)"
+
+
+function init_environment {
+    echo "machine github.com login $GITHUB_TOKEN password x-oauth-basic" >> ~/.netrc
+    chmod 0600 ~/.netrc
+
+    git config --global user.email "$ROBOT_EMAIL"
```

Comment:
I think it'd be best not to do any changes to the git global config. Can you just use `GIT_AUTHOR_NAME` / `GIT_AUTHOR_EMAIL` env variables + `--no-gpg-sign` when doing `git commit` ?

Reply:
honestly, I must have missed the existence of `--no-gpg-sign` in the git manpage. I'll of course use that, thanks!